tests/test_marshmallow_loader.py

```diff
@@ -86,6 +98,31 @@ def test_marshmallow_load_errors(app, db, es, test_data, search_url,
         assert res.status_code == 400
 
 
+def test_marshmallow_load_nested_errors(app, db, es, test_data, search_url,
+                                        search_class):
+    app.config['RECORDS_REST_DEFAULT_LOADERS'] = {
+        'application/json': loaders.marshmallow.marshmallow_loader(
+            _TestSchemaNested
+        ),
+        'application/json-patch+json': loaders.json_patch_v1,
+    }
+
+    with app.test_client() as client:
+        HEADERS = [
+            ('Accept', 'application/json'),
+            ('Content-Type', 'application/json')
+        ]
+
+        # Create record
+        missing_top_most_required_field = dict(nested="test")
```

Comment:
Just as a useless comment, I would use  `{'nested': 'test'}` to make it more readable as JSON is our internal representation, however, I see the dict constructor is used everywhere...